MetaMask/metamask-extension
# Remove MV3 specific account code in metamask-controller.js

## **Description**

Now that we no longer need to handle the service worker restart case, we don't need to use the MV3 specific keyring and account logic in the MetaMask controller. This also eliminates a bug where, on an MV3 build, when you create a new account you would not be switched to that account.

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Build on MV3. Create a new account, you should be switched to that account
2. Lock and unlock, it should work fine
3. Add multiple accounts, close the browser, open metamask, all accounts should still be there
4. Leave MetaMask running for 10 minutes, it should not lock

## **Screenshots/Recordings**

### **Before**

### **After**

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
